REFERENCE.md

```
@@ -7233,3 +7236,78 @@ if err := client.Auth().UpdateLoginUserDetails(ctx, req); err != nil {
 </dd>
 </dl>
 </details>
+
+## Events
+
+<details><summary><code>client.Events().<a href="https://github.com/scalekit-inc/scalekit-sdk-go/blob/main/events.go">ListEventsPaginated</a>(ctx, options) -> (*ListEventsPaginatedResponse, error)</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)